inc/cachify.class.php

```diff
@@ -1124,6 +1086,80 @@ private static function _is_logged_in() {
 		return false;
 	}
 
+	/**
+	 * Register all hooks to flush the total cache
+	 *
+	 * @since   2.4.0  Make the function public
```

Comment:
I'm not sure what you mean by that comment? It sounds more like `@todo` than `@since`.

Reply:
@krafit  added the related issue to milestore 2.4
That's why I added @since 2.4.0

What would you recommend?

Reply:
`@since` should store only the version or date since we have this method, var. A change should not a part of the doc-bloc, especially not the `@since` tag.

Reply:
So should I remove the `@since` tag completely or only the description?

Reply:
Only the description.

Reply:
@chesio  ok, removed it.

@bueltge don't know. I think the other open tasks on the 2.3.0 milestone are  much more complex than this PR. So including this PR into 2.3.0 should not delay the 2.3.0 release.

Reply:
@bueltge I added this to the `2.4.0` milestone because I did not expect this task to be fixed so quickly. But since @derweili did a stellar job here, we should move this to the `2.3.0` milestone and publish it as soon as possible.